src/components/table/table.js

```diff
@@ -274,6 +274,15 @@ class Table extends React.Component {
     caption: PropTypes.string,
 
     /**
+     * The HTML id of the element that contains a description
+     * of this table.
+     *
+     * @property ariaDescribedBy
+     * @type string
+     */
+    ariaDescribedBy: PropTypes.string,
```

Comment:
Do you think we should match the React API prop name `aria-describedby`?

Reply:
Yes i think its easier than converting all the tags